### DISCUSSION TITLE
[Feature Suggestion] Navigating huge mindmaps by replaceing object groups with one box as expand/collapse

### DISCUSSION THREAD

**Discussion by jonaseichhorst** [excalidraw/excalidraw#7905] [General]
Thanks firstly for a great tool! I enjoy it so much that actually I'm creating a quite huge whiteboard full of diagrams, scribbles etc that at some point become hard to navigate.

I was thinking therefore, if you could essentially name a group of objects that could either be collapsed into one bigger box or automatically replaced at a certain zoom level or so - it could be an interesting way to make it an inifinite sized digital whiteboard but still be able to effectivly navigate it and make the best of both the analog whiteboarding experience but also digital (collaborative & easier to navigate/dynamic expand/collapse).

Don't have skills/time to develop it as pull request myself unfortunately but happy to spend a bit more time sharpening the idea if anybody finds it useful.

Thanks!

**Comment by arminus** [comment]
Basically like draw.io container shapes

**Comment by sojusnik** [comment]
Another way would be using [links to objects](https://github.com/excalidraw/excalidraw/issues/3276) on the whiteboard.
